examples/light-switch-app/ameba/README.md

```
@@ -26,19 +26,19 @@ The CHIP demo application is supported on
 
 -   Pull docker image:
 
-          $ docker pull ghcr.io/project-chip/chip-build-ameba:200
+            $ docker pull ghcr.io/project-chip/chip-build-ameba:211
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)